app/scripts/lib/util.js

```diff
@@ -174,3 +174,15 @@ export {
   getChainType,
   checkAlarmExists,
 };
+
+// Taken from https://stackoverflow.com/a/1349426/3696652
+const characters =
```

Comment:
We already have `uuidv4` and are using for getting a random ID -- maybe we use that instead?

Reply:
yep we actually can, but I used this as requirement here is just any random number and not uuid